src/main/java/bio/terra/service/resourcemanagement/azure/AzureMonitoringService.java

```diff
@@ -74,9 +74,7 @@ public AzureMonitoringService(AzureResourceConfiguration resourceConfiguration)
   public Workspace getLogAnalyticsWorkspace(
       BillingProfileModel profileModel, AzureStorageAccountResource storageAccount) {
     LogAnalyticsManager client =
-        resourceConfiguration.getLogAnalyticsManagerClient(
-            resourceConfiguration.credentials().getHomeTenantId(),
-            profileModel.getSubscriptionId());
```

Comment:
Nice refactoring!

Reply:
Agreed! Thanks for catching that - this is much cleaner.